contracts/RiskFund/ProtocolShareReserve.sol

```diff
@@ -23,11 +23,11 @@ contract ProtocolShareReserve is Ownable2StepUpgradeable, ExponentialNoError, Re
 
     /**
      * @dev Initializes the deployer to owner.
-     * @param _protocolIncome  remaining protocol income.
+     * @param _protocolIncome The address, protocol income will be sent to.
```

Comment:
```suggestion
     * @param _protocolIncome The address protocol income will be sent to.
```

Reply:
Fixed , can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/151/commits/ffd493f85fd85b0e1ede68a5b5b8ca0e298698e4).